django/test/client.py

```diff
@@ -835,8 +835,11 @@ def _handle_redirects(self, response, data='', content_type='', **extra):
             if url.port:
                 extra['SERVER_PORT'] = str(url.port)
 
+            path = url.path
+            # RFC 2616: bare domains without path are treated as the root.
+            if not path and url.netloc:
+                path = '/'
```

Comment:
I thought this `path = url.path or url.netloc and '/'` was too hard to reason about. 

Then moving about relative to the old comment read clearer I thought... 

🤔

Reply:
Much better, thanks!

Reply:
OK, thanks. I'll squash it down then. 😄